---
layout: page
title: Research
---

### Research Interests

My graduate research (2013-2017) focused on the controllability of Boolean networks, theoretical models for Gene Regulatory Networks. It has significantly enhanced my understanding of genomics. My contributions to machine learning have led to new methods for disease diagnosis and prognoses. 
My postdoctoral training (2017-2019), under the joint mentorship of Drs. Chakravarti and Saria,
represented a first shift in focus from theoretical to computational genomics.
My current postdoctoral research (2019-now) with Drs. Ji, Hicks, and Feinberg has provided complementary expertise in statistics, single-cell genomics, epigenomics, and spatial transcriptomics. I am passionate about a variety of problems related to gene regulation, including genomics and epigenomics. I am focusing on developing computational and statistical methods for interpreting high-throughput data of multi modalities, especially single-cell data. 

Specifically, I am passionate about the following research topics. 

- **Methods**: developing mathematical, statistical, and machine learning methods to model the spatio-temporal patterns in single-cell and spatial genomics and epigenomics data, Bayesian statistics, functional data analysis, and deep neural networks.

- **Theoretical models**: gene regulatory networks (Boolean networks) controllability and inference, probability, and optimization.

- **Collaboration and scientific research**: glioblastoma, non-small cell lung cancer, head and neck squamous cell carcinoma, pancreatic cancer, immunology, infectious disease, obesity, and maternal and child health.




### Grants

 **NIH Pathway to Independence Award (1K99HG011468)**, NIH/NHGRI, 3/2021 - 3/2026

 Title: Computational Methods for Inferring Single-cell DNA Methylation and its Spatial Landscape

Role: principal investigator
